crates/sovereign-sdk/full-node/db/sov-db/src/rocks_db_config.rs

```diff
@@ -44,10 +44,33 @@ impl<'a> RocksdbConfig<'a> {
     /// Build [`rocksdb::Options`] from [`RocksdbConfig`]
     pub fn as_rocksdb_options(&self, readonly: bool) -> Options {
         let mut db_opts = Options::default();
+
+        let mut block_based_options = BlockBasedOptions::default();
+        /*
+         * The following settings are recommended in:
+         * https://github.com/facebook/rocksdb/wiki/memory-usage-in-rocksdb
+         */
+        // Since our database is write heavy, disable cache to decrease memory footprint
```

Comment:
are you sure about this?

when we get more usage (not like on our tests where we produce mostly empty blocks) reads will also be very common, especially when we start to get a lot of transactions

Reply:
Maybe i mis-worded the comment. But in general, disabling cache would remove the possibility of OOM errors.

Instead of doing that, i have set an LRU cache to a larger value (100 MB) rather than the default (8 MB). However, this would not remove the possibility of OOM errors.